packages/core/src/components/Cards/SuccessStoryCard/SuccessStoryCardMobile.js

```diff
@@ -8,7 +8,17 @@ import RemoveOrphans from '../../RemoveOrphans/RemoveOrphans';
 
 import css from './SuccessStoryCardMobile.css';
 
-const SuccessStoryCardMobile = ({title, imageSrc, copy, brands = [], href, brandsTitle, variant}) => (
+type Props = {
+  title: string,
+  imageSrc: string,
+  copy: string,
+  brands: Array<any>,
+  href: string,
+  variant?: string,
+  brandsTitle: string,
+}
+
+const SuccessStoryCardMobile = ({title, imageSrc, copy, brands = [], href, brandsTitle, variant}: Props) => (
```

Comment:
for consistency is it worth giving brands a default value in `defaultProps` instead?